### PR TITLE
improve code

### PR DESCRIPTION
**Pull request by joamatab**
- remove import *

### CODE REVIEW

File: meow/geometries.py [hunk 1]
```diff
@@ -153,7 +153,7 @@ def _mask2d_axis_x(self, X, Y, z):
                 return np.zeros_like(Y, dtype=bool)
             intersections = sg.MultiLineString([intersections])
 
-        mask = np.zeros_like(Y, dtype=bool)
+        np.zeros_like(Y, dtype=bool)
```
Comment (jan-david-fischbach) [inline meow/geometries.py:156]: I think we can fully leave this out

File: meow/eme/sax.py [hunk 2]
```diff
@@ -32,7 +32,7 @@ def _get_netlist(propagations, interfaces):
         **{k: S for k, S in interfaces.items()},
     }
 
-    interface_keys = list(interfaces)
+    list(interfaces)
```
Comment (jan-david-fischbach) [inline meow/eme/sax.py:35]: This line can also be omitted

File: meow/geometries.py (hunk 1 above)
Comment (flaport) [reply meow/geometries.py:156]: we can't. I'm not even sure why this line was changed. As you can see mask needs to be defined for the line

```python
            mask |= (x_min <= X) & (X <= x_max) & (y_min <= Y) & (Y <= y_max)
```

to work.

File: meow/eme/sax.py (hunk 2 above)
Comment (flaport) [reply meow/eme/sax.py:35]: I don't understand why this code 'improvement' tool not automatically removes the full line.

**Comment by flaport** [comment]
I don't know man. These are just aesthetic changes and it introduces a bug in geometries.py. I might add explicit imports in my __init__.py manually later on. Closing for now.

**Pull request closed without merging**